scheduler/scheduler-server/src/test/java/functionaltests/db/schedulerdb/TestReadSchedulerAccount.java

```diff
@@ -50,10 +50,21 @@
 
         int taskCount;
 
+        // TODO complete the singleJobScenario with task the following statistics
```

Comment:
I think it is better to create a task than to keep a TODO in the code

Reply:
Sure, the todo was just for during the review to avoid getting asked to remove unused variables. I'll complete the test before merging